app/src/main/java/com/parasoft/demoapp/HomeActivity.java

```diff
@@ -107,6 +108,9 @@ public void onResponse(@NonNull Call<ResultResponse<OrderListResponse>> call,
                         if (orderList == null || orderList.size() == 0) {
                             showNoOrderView();
                         } else {
+                            for (OrderResponse orderResponse : orderList) {
+                                orderResponse.setSubmissionDate(TimeUtil.convertToLocalTime(orderResponse.getSubmissionDate()));
```

Comment:
Instead of modifying the "OrderResponse" object, please move the util call inside "OrderAdapter.onBindViewHolder" to get a string of converted time right before setting text fields.